config/ufs/machines/config_machines.xml

```diff
@@ -451,7 +451,77 @@ This allows using a different mpirun command to launch unit tests
       <env name="W3NCO_LIBd">$ENV{NCEPLIBS_DIR}/lib/libw3nco_v2.0.6_d.a</env>
     </environment_variables>
   </machine>
-
+ <machine MACH="gaea">
+    <DESC>
+      port to gaea
+    </DESC>
+    <NODENAME_REGEX>gaea*</NODENAME_REGEX>
+    <OS>CNL</OS>
+    <COMPILERS>cray-intel</COMPILERS>
+    <MPILIBS>mpich</MPILIBS>
+    <PROJECT>esrl</PROJECT>
+    <SAVE_TIMING_DIR> </SAVE_TIMING_DIR>
+    <CIME_OUTPUT_ROOT>$ENV{SCRATCH}/$ENV{USER}/ufs-runs/scratch</CIME_OUTPUT_ROOT>
+    <DIN_LOC_ROOT>$ENV{SCRATCH}/$ENV{USER}/ufs-runs/inputdata</DIN_LOC_ROOT>
+    <DIN_LOC_ROOT_CLMFORC>$ENV{SCRATCH}/$ENV{USER}/ufs-runs/inputdata/lmwg</DIN_LOC_ROOT_CLMFORC>
+    <DOUT_S_ROOT>$ENV{SCRATCH}/$ENV{USER}/ufs-runs/archive/$CASE</DOUT_S_ROOT>
+    <BASELINE_ROOT>$ENV{SCRATCH}/$ENV{USER}/ufs-runs/baselines</BASELINE_ROOT>
+    <CCSM_CPRNC>$ENV{SCRATCH}/$ENV{USER}/ufs-runs/tools/cprnc/cprnc</CCSM_CPRNC>
+    <GMAKE>make</GMAKE>
+    <GMAKE_J>8</GMAKE_J>
+    <BATCH_SYSTEM>slurm</BATCH_SYSTEM>
+    <SUPPORTED_BY>mpotts@redlineperf.com</SUPPORTED_BY>
+    <MAX_TASKS_PER_NODE>16</MAX_TASKS_PER_NODE>
+    <MAX_MPITASKS_PER_NODE>16</MAX_MPITASKS_PER_NODE>
+    <PROJECT_REQUIRED>FALSE</PROJECT_REQUIRED>
+    <mpirun mpilib="default">
+     <executable>srun</executable>
+      <arguments>
+        <arg name="label"> --label</arg>
+        <arg name="num_tasks"> -n {{ total_tasks }}</arg>
+    </arguments>
+    </mpirun>
+    <module_system type="module" allow_error="true">
```

Comment:
Do you know that we need the allow_error="true" option - lets not use it if we don't need to.  (Some module setups write info messages to stderr - that's when we need this flag)